### ISSUE TITLE
Dark theme

### ISSUE THREAD

**Issue by rakleed** [mozilla/kitsune#5326]
Could you please add a dark theme to the site? Because using your site at night or in dark rooms is very unpleasant because of the bright white background.

**Comment by dannycolin** [comment]
Duplicate of #4662

**Comment by dannycolin** [comment]
This is blocked by [mozilla/protocol#647](https://github.com/mozilla/protocol/issues/647)

**Comment by anewuser** [comment]
@dannycolin That issue has been closed by a designer. Please explain to the design team that this is a security issue. People who use extensions like Dark Reader are told to clear `extensions.webextensions.restrictedDomains` in `about:config` to be able to make the background of AMO and SUMO dark.

**Comment by denyshon** [comment]
Hi, does anyone happen to know the current state of the issue? @emilghittasv (sorry if it's a wrong mention, but I've seen you dealing with the issues...)

**Comment by emilghitta** [comment]
Hi @denyshon 👋 

The dark theme implementation is not currently under the engineering team's radar/not currently being worked on due to some more urgent priorities.

**Comment by kelimuttu** [comment]
See also: https://bugzilla.mozilla.org/show_bug.cgi?id=1761983
